### ISSUE TITLE
Colorbot not working

### ISSUE THREAD

**Issue by Veeeeeedi** [Seconb/Arsenal-Colorbot#14]
It was working perfectly fine yesterday bt i have no cle why it isnt working today
any tips?

**Comment by Seconb** [comment]
need more info

**Comment by Seconb** [comment]
also my next response will be slow (8 hours maybe if i don’t forget to respond) cuz i’m busy

tell me when the error happens and all that

**Comment by Veeeeeedi** [comment]
It shows no error per say it just doesnt lock on to the enemy anymore
basically its like playing arsenal normally aka without hacks

**Comment by Veeeeeedi** [comment]
yeah i just went through that, i wasnt using fullscreen so i tried it but it still didnt work :(

**Comment by Veeeeeedi** [comment]
so i tried some other options sch as aimmy and nezure but none of them worked as well
so its probably something in roblox itself which i have messed up, i honestly have no clue whats going on
